tests/integration/suite/test_notifier.py

```diff
@@ -0,0 +1,20 @@
+from .common import random_str
+
+
+def test_notifier_wechat_apiurl(admin_mc, remove_resource):
+    client = admin_mc.client
+    name = random_str()
+    n = client.create_notifier(
+        clusterId="local",
+        name=name,
+        wechatConfig={"agent": "1111",
+                      "apiUrl": "test.com",
+                      "corp": "corp",
+                      "defaultRecipient": "2",
+                      "recipientType": "party",
+                      "secret": "231"})
+    remove_resource(n)
+    assert n is not None
+    assert (n["wechatConfig"]["apiUrl"]) == "test.com"
+    n = client.update(n, wechatConfig={"apiUrl": "test2.com"})
```

Comment:
what is the point of this part? just that it can be updated? We should probably test the default url too. I think maybe we can test the default url, then update it to a custom url and test that.